govlearn-api/src/main/java/com/unimuenster/govlearnapi/tags/service/TagsService.java

```diff
@@ -24,6 +28,9 @@ public class TagsService {
     private final TagRepository tagRepository;
     private final ServiceTagMapper serviceTagMapper;
     private final EntityManager entityManager;
+    private final CourseRepository courseRepository;
+    /** evtl. unsauber hier Kurs-Klassen aufzurufen, jedoch wusste ich nicht wie ich
+    von CourseDTO zu Course casten kann, was wohl nötig wäre um das zu umgehen **/
 
```

Comment:
Einfach einen neuen Mapper dafür schreiben in diesem Package.

Reply:
verstehe ich nicht ganz, meinst du in ControllerCourseMapper?

Reply:
Nee würde hier noch einen ContollerTagsMapper erstellen und das da rein schreibe, weil das hier ein anderer Use Case ist.

Reply:
musst du mir morgen mal erklären